code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -139,24 +139,30 @@
 
 
 /mob/living/carbon/xenomorph/adjustBruteLoss(amount, updating_health = FALSE)
+	if(stat == DEAD)  //Dead benos take no further damage.
+		return
```

Comment:
Why

Reply:
Why have dead benos continue with the proc and take up more resources?

Reply:
remove it